libethashseal/GenesisInfo.h

```diff
@@ -67,6 +67,8 @@ enum class Network
     ConstantinopleNoProofTest = 82,
     /// Byzantium + Constantinople + ConstantinopleFix active from block 0
     ConstantinopleFixTest = 83,
+    /// ConstantinopleFixTest + Istanbul active from block 2
```

Comment:
Why is this active from block 2 rather than block 0?

Reply:
This is a special kind of config for the tests to be able to check the behavior before Istanbul (in block 1) as well as after Istanbul. We have similar configs for other forks, too.